### PR TITLE
Jlab newsletter

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
I resurrected the newsletter series (now described as "occasional"!) and got a newsletter for JLab ready (a few people reviewed the content).

I added quite a few photos that makes the newsletter look rather nice (and reorganised the images directory a bit).

Please squash on merge!

### CODE REVIEW

**Comment by jouvin** [comment]
@graeme-a-stewart Thanks for doing this and sorry for not being able to comment the early draft... I'm reviewing your PR... (minor) Comments should be available soon!

File: newsletter/_posts/2019-04-01-Jlab-Workshop.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: HSF-OSG-WLCG Workshop at Jefferson Lab, HOW2019
+author: Graeme Stewart
+layout: newsletter
+---
+
+The annual [HEP Software Foundation
+workshop](https://indico.cern.ch/event/759388/){:target="jlab_agenda"} happened
+this year in [Jefferson
```
Comment (jouvin) [inline newsletter/_posts/2019-04-01-Jlab-Workshop.md]: Remove 'this year' as it is rather obvious and is being repeated at the beginning of next sentence.

File: newsletter/_posts/2019-04-01-Jlab-Workshop.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+already have, received wide support, but the details of how to manage this, for
+all communities, need to be worked out.
+
+<div style="text-align:center; padding:25px; float:right">
+<img src ="/images/workshops/jlab-plenary1.jpg" alt="JLab Workshop Group Photo" />
+<br>Plenary Discussions. Photo &copy; DOE Jefferson Laboratory
+</div>
+
+Common sessions for HSF and WLCG on Tuesday looked at the evolution of
+technology - processors, storage and networking - and how HEP is managing to
```
Comment (jouvin) [inline newsletter/_posts/2019-04-01-Jlab-Workshop.md]: After 'networking -', I'd add 'based on the impressive work done by the [HEPiX Benchmarking WG](http://w3.hepix.org/benchmarking.html)'

File: newsletter/_posts/2019-04-01-Jlab-Workshop.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+On Friday the sessions turned back to plenary mode and we heard from projects
+being funded to provide the investment in software and computing that we so very
+much need. It was therefore very appropriate to announce that the HSF Community
+White Paper Roadmap was finally
+[published](https://doi.org/10.1007/s41781-018-0018-8){:target="cwp_roadmap_csbs"}
+in *Computing and Software for Big Science* during the week of the workshop.
+
+That led us very neatly to a closing talk from JLab's Amber Boehnlein, on
+her thoughts about the future of computing the field. Amber was the main
```
Comment (jouvin) [inline newsletter/_posts/2019-04-01-Jlab-Workshop.md]: 'the field' -> 'in the field'

File: newsletter/_posts/2019-04-01-Jlab-Workshop.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+of March. This year we had the opportunity to join forces again with the
+Worldwide LHC Computing Grid (WLCG) community and the  US Open Science Grid
+(OSG). Almost 250 scientists, from LHC, HEP and non-HEP communities joined the
+meeting.
+
+On the first day we opened the meeting with an excellent introduction to JLab
+from lab director Stuart Henderson and an overview of JLab computing and
+software from Raffaella De Vita. That was followed by plenary talks from the LHC
+experiments, other HEP experiments and many non HEP communities, including the
+next generation US nuclear physics experiment, the Electron-Ion Collider. That
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2019-04-01-Jlab-Workshop.md]: maybe experiment -> project?

File: newsletter/_posts/2019-04-01-Jlab-Workshop.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+session that explored our links with the wider Python community, with an
+emphasis on toolset approaches where different tools mesh together to form the
+required pipeline. There was also a presentation from outside HEP, with Jonathan
+Helmus from Anaconda introducing the numba Python JIT and the Conda package
+distribution. In the latter our own community has contributed ROOT on Linux and
+OS X platforms, which is already very popular.
+
+The theme of packaging was touched on again in the [*Software Development
+Tools*](/workinggroups/softwaredevelopertools.html){:target="wg_swtools"}
+session. The HSF Packaging WG presented solutions that support the wider science
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2019-04-01-Jlab-Workshop.md]: You forgot a link for you favourite WG ;-).

**Comment by graeme-a-stewart** [comment]
Thanks for the careful reading and suggestions. All fixed now.

The CI is failing on https://nixcon2018.org/ and has been since last night (at least). We might have to disable that link.

**Comment by jouvin** [comment]
LGTM... Merging...

**Pull request merged**